package.json

```diff
@@ -72,7 +73,8 @@
     "@commitlint/cli": "^17.6.3",
     "@commitlint/config-conventional": "^17.6.3",
     "@dlgshi/cypress-plugin-designtokens": "^0.0.1-development",
-    "@playwright/experimental-ct-react17": "^1.36.1",
+    "@playwright/experimental-ct-react17": "~1.40.0",
+    "@playwright/test": "~1.40.0",
```

Comment:
**note**: Playwright has changed the peer dependencies of the component testing library in version 1.38, so we need to install `@playwright/test` separately.